deploy/examples/crds.yaml

```diff
@@ -2280,6 +2280,8 @@ spec:
                   x-kubernetes-validations:
                     - message: at least one network selector must be specified when using multus
                       rule: '!has(self.provider) || (self.provider != ''multus'' || (self.provider == ''multus'' && size(self.selectors) > 0))'
+                    - message: the legacy hostNetwork settingcan only be set if the network.provider is set to the empty string
```

Comment:
nit: typo "settingcan"

Reply:
@rkachach  wrote:
> nit: typo "settingcan"

already spotted and fixed.